src/pages/VideoListingPage/VideoListingPage.css

```diff
@@ -0,0 +1,54 @@
+
+.videolist-container{  justify-content: center; }
+
+
+.categories-batch{
+    cursor: pointer;
+    padding: 0.5em;
+    margin: 0.5em;
+    background-color: var(--boldy-light-purple);
+    background-color: var(--boldy-dark-purple);
+    color:white;
+    border-radius: 1rem;
```

Comment:
Use consistent unit or variables, and using rem for border-radius could cause an issue if the root font side change. So try to use px for border-radius.

Reply:
done